products/nextjs.md

```diff
@@ -59,9 +59,9 @@ releases:
 > developers to build performant and scalable web applications. The framework is designed to be
 > developer-friendly, with a focus on fast refresh and an optimized production build.
 
-Next.js is following [semantic versioning](https://semver.org/), but does not have a clearly defined
-release and support policy. Looking at the latest releases, it seems only the latest major release
-is supported with mainly bug and security fixes.
+Next.js follows [semantic versioning](https://semver.org/). Major versions are released on average twice per year,
+minor versions more frequently, and patch versions very frequently. Importany security patches can be 
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Typo

```suggestion
minor versions more frequently, and patch versions very frequently. Important security patches can be 
```